src/app/pages/listings/contents/listings-content/m.ts

```diff
@@ -0,0 +1,86 @@
+/**
```

Comment:
Filename m.ts these are utils use name utils or helpers

Reply:
This is some left over file. I have deleted.